bytecomp/bytelink.ml

```diff
@@ -608,24 +619,25 @@ let link ppf objfiles output_name =
       let exec_name = fix_exec_name output_name in
       if not (build_custom_runtime prim_name exec_name)
       then raise(Error Custom_runtime);
-      if !Clflags.make_runtime
-      then (remove_file bytecode_name; remove_file prim_name)
-      else append_bytecode_and_cleanup bytecode_name exec_name prim_name
+      if !Clflags.make_runtime then begin
+        remove_file bytecode_name;
+        if not !Clflags.keep_camlprimc_file then remove_file prim_name
+      end else append_bytecode_and_cleanup bytecode_name exec_name prim_name
     with x ->
       remove_file bytecode_name;
-      remove_file prim_name;
+      if not !Clflags.keep_camlprimc_file then remove_file prim_name;
       raise x
   end else begin
     let basename = Filename.chop_extension output_name in
     let temps = ref [] in
-    let c_file =
+    let c_file, stable_name =
```

Comment:
It seems to me that `stable_name` should be an option

Reply:
Possibly, however it is not an option in `build_custom_runtime`
(one can only avoid passing `-fdebug-prefix-map` by keeping
the generated file).